metrics_activities_exports.go

```diff
@@ -0,0 +1,50 @@
+package chartmogul
+
+// MetricsActivitiesExport represents Metrics API activity export in ChartMogul.
+type MetricsActivitiesExport struct {
+	ID        string `json:"id"`
+	Status    string `json:"status"`
+	FileURL   string `json:"file_url"`
+	Params    Params `json:"params"`
+	ExpiresAt string `json:"expires_at"`
+	CreatedAt string `json:"created_at"`
+}
+
+type Params struct {
+	Kind   string       `json:"kind"`
+	Params NestedParams `json:"params,omitempty"`
+}
+
+type NestedParams struct {
+	ActivityType string `json:"activity_type,omitempty"`
+	StartDate    string `json:"start_date,omitempty"`
+	EndDate      string `json:"end_date,omitempty"`
+}
+
+// NewMetricsActivitiesExport is the POST-ed to create a MetricsActivitiesExport .
+type NewMetricsActivitiesExport struct {
+	Type      string `json:"type,omitempty"`
+	StartDate string `json:"start-date,omitempty"`
```

Comment:
is it `start-date` or `start_date`?

Reply:
The params on the returned JSON use `start_date`, but for compatibility with existing metrics API we use the `start-date` as params (eg https://dev.chartmogul.com/reference#endpoint-overview-metrics-api )